src/styles/index.css

```diff
@@ -0,0 +1,15 @@
+@import url("https://fonts.googleapis.com/css2?family=Cantarell:ital@0;1&display=swap");
+
+body {
+  margin: 3;
+  background-color: #fab0b0;
+  font-family: -apple-system, Cantarell, sans-serif;
+  background-image: url(https://www.aboutmanchester.co.uk/wp-content/uploads/2017/11/L.S.-Lowry-The-Rush-oil-on-canvas-1964-est.-%C2%A3800000-1200000.jpg);
+  /*font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;*/
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
```

Comment:
This can be removed as the `code` element is never used in the app

Reply:
Removed.